homeassistant/components/hive/binary_sensor.py

```diff
@@ -1,28 +1,29 @@
 """Support for the Hive binary sensors."""
-from homeassistant.components.binary_sensor import (
-    DEVICE_CLASS_MOTION,
-    DEVICE_CLASS_OPENING,
-    BinarySensorEntity,
-)
+from datetime import timedelta
+
+from homeassistant.components.binary_sensor import BinarySensorEntity
 
 from . import DATA_HIVE, DOMAIN, HiveEntity
 
-DEVICETYPE_DEVICE_CLASS = {
-    "motionsensor": DEVICE_CLASS_MOTION,
-    "contactsensor": DEVICE_CLASS_OPENING,
+DEVICETYPE = {
+    "contactsensor": {"type": "opening"},
```

Comment:
Please use our device class constants imported from the binary sensor integration.